workflow/layout/hofx_only.yaml

```diff
@@ -0,0 +1,48 @@
+suite: !Cycle
+      <<: *suite_defaults
+
+      obs_prep: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  ( ~ suite.has_cycle('-24:00:00') | hofx_run.at('-24:00:00'))
+        resources: !calc partition.resources.run_obs_prep
+        config: [ base ]
+        J_JOB: JJOB_OBS_PREP
+
+      hofx_prep: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  obs_prep
+        resources: !calc partition.resources.run_fcst_prep
+        config: [ base ]
+        J_JOB: JJOB_HOFX_PREP
+
+      da_prep_soca: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  hofx_prep
+        resources: !calc partition.resources.run_3dvar
+        config: [ base ]
+        J_JOB: JJOB_PREP_SOCA
+
+      hofx_run: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  da_prep_soca
+        resources: !calc partition.resources.run_3dvar
+        config: [ base ]
+        J_JOB: JJOB_HOFX
+
+      post: !Task
+        <<: *shared_task_template
+        Trigger: !Depend  hofx_run
+        resources: !calc partition.resources.run_fcst_prep
+        rocoto_command: "echo post"
```

Comment:
Remove the  rocoto_command: "echo post" (see 3dvar case) and check the figures in /Figures